seek-n-major.json

```diff
@@ -0,0 +1,92 @@
+{
+  "extends": [
+    ":prHourlyLimit4",
+    ":rebaseStalePrs",
+    ":renovatePrefix",
+    ":semanticCommits",
+    ":timezone(Australia/Melbourne)",
+    ":updateNotScheduled",
+    "preview:buildkite",
+    "preview:dockerCompose",
+    "docker:disableMajor"
+  ],
+  "packageRules": [
+    {
+      "depTypeList": ["dependencies"],
+      "semanticCommitType": "fix"
+    },
+    {
+      "commitMessageExtra": "{{newValue}}",
+      "commitMessageTopic": "{{depName}}",
+      "matchManagers": ["buildkite", "gomod", "npm", "nvm"]
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["^@types/"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "depTypeList": ["devDependencies"],
+      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
+      "excludePackagePatterns": ["^@?seek", "seek$", "^@types/"],
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on Tuesday"
+    },
+    {
+      "depTypeList": ["peerDependencies"],
+      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
+      "excludePackagePatterns": ["^@?seek", "seek$", "^@types/"],
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["major"],
+      "schedule": "after 3am and before 6am on Tuesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "excludePackagePatterns": ["^seek-jobs/", "^seek-oss/"],
+      "groupName": "buildkite plugins",
+      "managerBranchPrefix": "",
+      "matchManagers": ["buildkite"],
+      "recreateClosed": true,
+      "schedule": "after 3am and before 6am on Wednesday"
+    },
+    {
+      "commitMessageExtra": "",
+      "group": {
+        "commitMessageTopic": "{{groupName}}"
+      },
+      "groupName": "docker images",
+      "matchManagers": ["docker-compose", "dockerfile"],
+      "recreateClosed": true,
+      "schedule": "after 3am and before 6am on Wednesday"
+    },
+    {
+      "matchPackageNames": ["braid-design-system", "sku", "skuba"],
+      "matchPackagePatterns": ["^@?seek", "seek$"],
+      "prPriority": 98,
+      "schedule": "after 3am and before 6am on every weekday"
+    },
+    {
+      "extends": ["group:monorepos"],
+      "excludePackageNames": [],
+      "recreateClosed": true
+    },
+    {
+      "automerge": false,
```

Comment:
Is there a reason why this has been flipped to `false`?

Reply:
Don't want to automerge majors?

Reply:
This rule isn't for majors though, it's for the `Pin dependencies` PR that tightens up e.g. tildes and carets for an application's dependencies.

Reply:
Oh right, me silly! My bad